ocaml/quicktest/quicktest.ml

```diff
@@ -878,6 +879,7 @@ let _ =
   finally
     (fun () ->
        (try
+          maybe_run_test "reconfigure-ip-cluster" (fun () -> Quicktest_ca282006.test s);
```

Comment:
Could this go in the same file as the other clustering tests?

Reply:
Technically yes, but I think we tend to keep quicktests in the quicktest folder by convention? If there are any other clustering quicktests, I could add them to the same file.

Reply:
quicktest_cluster.ml would be fine, you are right they are different from unit tests, and this is the first clustering quicktest.